R/mutate.R

```diff
@@ -23,10 +77,7 @@ salsa <- function(.tbl, ..., .env = caller_env()) {
   as_tibble_splice(results)
 }
 
-#' chacha
-#'
-#' @param .tbl TODO
-#'
+#' @rdname samba
```

Comment:
Using `#' @rdname samba` to make sure these four related functions are documented together.